src/css/components/panel.scss

```diff
@@ -80,6 +91,12 @@ h4 + .panel {
     margin: 1rem -1rem -1rem;
     padding: 1rem;
     width: 100%;
+
+    .route-form-preview {
```

Comment:
This should actually go in the dashboard code:

https://github.com/18F/cg-dashboard/blob/067ef8fe56532ed2e032a18be9ad53b357d11586/static_src/css/route_form.css

I mean, technically it should go in cg-style, but we don't have a place for it yet, so it would do better there for now.